x-pack/plugins/siem/common/endpoint/types.ts

```diff
@@ -6,11 +6,17 @@
 
 import { Datasource, NewDatasource } from '../../../ingest_manager/common';
 
+export interface AppLocationState {
+  isTabChange?: boolean;
+  prevPath?: string;
+}
+
 export interface AppLocation {
   pathname: string;
   search: string;
   hash: string;
   key?: string;
+  state?: AppLocationState;
```

Comment:
:grey_question: A comment about when you would/wouldn't include this in an AppLocation might be helpful to the reader.

Reply:
will do